src/Arriba/Arriba.Server/Extensions/OAuthExtension.cs

```diff
@@ -28,9 +32,28 @@ public static IServiceCollection AddOAuth(this IServiceCollection services, IArr
                 auth.DefaultPolicy = jwtBearerPolicy;
             });
 
-            services.AddSingleton((_) => serverConfig.OAuthConfig);
```

Comment:
Is this getting registered somewhere else or was it a duplicate registration? Seems like stuff would break with this change

Reply:
It is registered at the startup and because of it I removed from the extension